exec/java-exec/src/main/java/org/apache/drill/exec/planner/logical/DrillConstExecutor.java

```diff
@@ -98,11 +98,37 @@
       //      - CHAR, SYMBOL, MULTISET, DISTINCT, STRUCTURED, ROW, OTHER, CURSOR, COLUMN_LIST
       .build();
 
+  private static ImmutableMap<SqlTypeName, TypeProtos.MinorType> CALCITE_TO_DRILL_MAPPING =
```

Comment:
Can you move this and the above type mapping to another class? Maybe a utility class.